controllers/PageController.php

```diff
@@ -72,7 +73,7 @@ public function actionView(int $id = null, $revisionId = null)
             $page->delete();
             throw new HttpException(404, 'Wiki page revision not found!');
         }
-
+        $page->doneEditing();
```

Comment:
We should not clear the status when viewing the page. It is quite possible that the user has two tabs open. One for editing and one to view the latest status.